Add keymaps to go to next/prev line; Change keymaps to be more vim-like

Lxtharia opened the pull request:
Hey, really love this project!

I changed some keymaps and added keymaps to move the selection up and down.
Makes syncing way more efficient!

New Keymaps:
J/K to move the line selection down/up
H/L to seek back/forwards
Space for play/pause as is common in video players
P for syncing as it's easily reachable with the pinkie

Lxtharia committed:
Add function to go to next/prev line; Make keymaps more vim-like

Using J/K to move the line selection and H/L to skip back/forward
Space for play/pause as is common in video players
P for syncing as it's easily reachable with the pinky

DBKarman commented:
Hi there,

Thank you for the kind words :)

I will review the changes sometime today or tomorrow, it looks good at a first glance. I've never used VIM so I didn't know it had line skipping functionality.

Props to you for going through my messy highschool code and adding the changes, even I have almost forgotten how the whole thing... just works.

Kind regards,
DBK

DBKarman committed:
Changed hint language to reflect changes in button mapping

- Changed hint to reflect button changes
- Changed the function of the UP-ARROW and DOWN-ARROW keys on the 4th screen from scroll up and down to go up a line or go down a line
